client/signup/steps/mlb-domains/index.jsx

```diff
@@ -0,0 +1,110 @@
+/**
+ * External dependencies
+ */
+var React = require( 'react' );
+
+/**
+ * Internal dependencies
+ */
+var StepWrapper = require( 'signup/step-wrapper' ),
+	Input = require( 'components/forms/form-text-input-with-affixes' ),
+	SelectDropdown = require( 'components/select-dropdown' ),
+	productsList = require( 'lib/products-list' )(),
+	Card = require( 'components/card' ),
+	SimpleNotice = require( 'notices/simple-notice' ),
+	SignupActions = require( 'lib/signup/actions' );
+
+module.exports = React.createClass( {
+	displayName: 'MlbDomainsStep',
+
+	getInitialState: function() {
+		return { blogName: '' };
```

Comment:
given you send this to `submitSignupStep` as `siteUrl`, I'd use that name here too for clarity.